src/components/Composer/index.android.tsx

```diff
@@ -1,100 +1,91 @@
+import {StyleProp, StyleSheet, TextInput, TextStyle} from 'react-native';
 import RNTextInput from '@components/RNTextInput';
 import * as ComposerUtils from '@libs/ComposerUtils';
 import themeColors from '@styles/themes/default';
+import {TextSelection} from './types';
 
-const propTypes = {
+type ComposerProps = {
     /** Maximum number of lines in the text input */
-    maxLines: PropTypes.number,
+    maxLines: number;
 
     /** If the input should clear, it actually gets intercepted instead of .clear() */
-    shouldClear: PropTypes.bool,
-
-    /** A ref to forward to the text input */
-    forwardedRef: PropTypes.func,
+    shouldClear: boolean;
 
     /** When the input has cleared whoever owns this input should know about it */
-    onClear: PropTypes.func,
+    onClear: () => void;
 
     /** Set focus to this component the first time it renders.
      * Override this in case you need to set focus on one field out of many, or when you want to disable autoFocus */
-    autoFocus: PropTypes.bool,
+    autoFocus: boolean;
 
     /** Prevent edits and interactions like focus for this input. */
-    isDisabled: PropTypes.bool,
+    isDisabled: boolean;
 
     /** Selection Object */
-    selection: PropTypes.shape({
-        start: PropTypes.number,
-        end: PropTypes.number,
-    }),
+    selection: TextSelection;
 
     /** Whether the full composer can be opened */
-    isFullComposerAvailable: PropTypes.bool,
+    isFullComposerAvailable: boolean;
 
     /** Allow the full composer to be opened */
-    setIsFullComposerAvailable: PropTypes.func,
+    setIsFullComposerAvailable: () => void;
 
     /** Whether the composer is full size */
-    isComposerFullSize: PropTypes.bool,
+    isComposerFullSize: boolean;
 
     /** General styles to apply to the text input */
-    // eslint-disable-next-line react/forbid-prop-types
-    style: PropTypes.any,
-};
-
-const defaultProps = {
-    shouldClear: false,
-    onClear: () => {},
-    autoFocus: false,
-    isDisabled: false,
-    forwardedRef: null,
-    selection: {
-        start: 0,
-        end: 0,
-    },
-    maxLines: undefined,
-    isFullComposerAvailable: false,
-    setIsFullComposerAvailable: () => {},
-    isComposerFullSize: false,
-    style: null,
+    style: StyleProp<TextStyle>;
 };
 
-function Composer({shouldClear, onClear, isDisabled, maxLines, forwardedRef, isComposerFullSize, setIsFullComposerAvailable, ...props}) {
-    const textInput = useRef(null);
+function Composer(
+    {
+        shouldClear = false,
+        onClear = () => {},
+        isDisabled = false,
+        maxLines,
+        isComposerFullSize = false,
+        setIsFullComposerAvailable = () => {},
+        style,
+        autoFocus = false,
+        selection = {
+            start: 0,
+            end: 0,
+        },
+        isFullComposerAvailable = false,
+        ...props
+    }: ComposerProps,
+    ref: ForwardedRef<TextInput>,
+) {
+    const textInput = useRef<TextInput>();
```

Comment:
```suggestion
    const textInput = useRef<TextInput>(null);
```

Let's bring back the `null`.

Reply:
Done!